robottelo/test.py

```diff
@@ -18,8 +18,11 @@
 from datetime import datetime
 from fabric.api import execute, settings
 from robottelo.cli.metatest import MetaCLITest
+from robottelo.cli.org import Org as OrgCli
```

Comment:
Is it reasonable to import `robottelo.cli` and call `cli.org.Org`? Similarly, it is reasonable to import `robottelo.ui` and call `ui.org.Org`?